wiki/Ranking_Criteria/Skinning/en.md

```diff
@@ -0,0 +1,331 @@
+Skin Set List 
+============= 
+
+The following table contains all skin sets that are presented in the draft worked out by the skinning subdivision of the [UBKRC-Project](https://osu.ppy.sh/forum/t/480331) (name credits go to Ephemeral)
+
+## osu! Cursor Set
```

Comment:
Just to ask, does the headings with prefixing game modes need to to prefix the game mode?
If so, I suggest making the game mode a level 2 followed by the set which would be a level 3 heading

```md
## osu!

### Cursor Set

[...]

## osu!taiko

### Hitobject Set

[...]

```